src/devices/CharacterLcd/README.md

```diff
@@ -23,14 +23,15 @@ Grove LCD RGB Backlight uses two i2c devices:
 - the device to control LCD (address 0x3E)
 - the device to control RGB backlight (address 0x62)
 
+(Make sure the Grove-LCD RGB Backlight is connected to a I2C port. Not the Digital port!)
```

Comment:
NIT: Can you remove parenthesis here? Seems like because we have the parenthesis this line and the next are being displayed together so removing it should fix it.

Reply:
Done 😊